cmd/controller-manager/app/controllermanager.go

```diff
@@ -269,6 +269,7 @@ func startNamespaceController(ctx controllerscontext.Context) (enabled bool, err
 		Client:                       ctx.Mgr.GetClient(),
 		EventRecorder:                ctx.Mgr.GetEventRecorderFor(namespace.ControllerName),
 		SkippedPropagatingNamespaces: skippedPropagatingNamespaces,
+		SkippedResourceConfig:        ctx.Opts.SkippedResourceConfig,
```

Comment:
For the namespace controller which really cares about is the namespaces it should be ignored, so how about merging the namespaces that specified in `--skipped-propagating-apis` and `--skipped-propagating-namespaces` to `skippedPropagatingNamespaces`? 

In other words, the namespaces controller don't have to know `SkippedResourceConfig`.